examples/computer_vision/iris_tf_keras/model_def.py

```diff
@@ -48,31 +43,33 @@ class IrisTrial(keras.TFKerasTrial):
     def __init__(self, context: keras.TFKerasTrialContext) -> None:
         self.context = context
 
-    def build_model(self) -> Model:
+    def build_model(self) -> models.Model:
         """
         Define model for iris classification.
 
         This is a simple model with one hidden layer to predict iris species (setosa, versicolor, or
         virginica) based on four input features (length and width of sepals and petals).
         """
-        inputs = Input(shape=(4,))
-        dense1 = Dense(self.context.get_hparam("layer1_dense_size"))(inputs)
-        dense2 = Dense(NUM_CLASSES, activation="softmax")(dense1)
+        inputs = layers.Input(shape=(4,))
+        dense1 = layers.Dense(self.context.get_hparam("layer1_dense_size"))(inputs)
+        dense2 = layers.Dense(NUM_CLASSES, activation="softmax")(dense1)
 
         # Wrap the model.
-        model = self.context.wrap_model(Model(inputs=inputs, outputs=dense2))
+        model = self.context.wrap_model(models.Model(inputs=inputs, outputs=dense2))
 
         # Create and wrap the optimizer.
-        optimizer = RMSprop(
-            lr=self.context.get_hparam("learning_rate"),
-            decay=self.context.get_hparam("learning_rate_decay"),
+        lr_schedule = optimizers.schedules.ExponentialDecay(
+            initial_learning_rate=self.context.get_hparam("learning_rate"),
+            decay_steps=10000,
```

Comment:
where did this value come from?

Reply:
It's a necessary argument. [Example code tends to just use a large-ish number](https://keras.io/api/optimizers/learning_rate_schedules/exponential_decay/).
I can take the magic number out of the Python code and put it in the experiment config along with the initial rate and decay.